apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -162,6 +205,10 @@ class PageTemplateModal extends Component {
 		trackSelection( this.props.segment.id, this.props.vertical.id, slug );
 		this.props.saveTemplateChoice( slug );
 
+		if ( this.props.isPostTitleBlockRegistered ) {
+			unregisterBlockType( 'a8c/spt-template-post-title' );
```

Comment:
As above. Should we also be tying the registration of this block to the component mount hook?